MiniApp/Classes/MiniApp.swift

```diff
@@ -89,9 +90,6 @@ public class MiniApp: NSObject {
     ///                         to interact with View component of mini app.
     ///         -   Error: Error details if Mini App View creating is failed
     ///   - messageInterface: Protocol implemented by the user that helps to communicate between Mini App and native application
-    @available(*, deprecated,
-    message:"Since version 2.0, you can create a Mini app view using just the mini app id",
-    renamed: "create(appId:completionHandler:messageInterface:)")
     public func create(appInfo: MiniAppInfo, completionHandler: @escaping (Result<MiniAppDisplayProtocol, Error>) -> Void, messageInterface: MiniAppMessageDelegate) {
```

Comment:
On Android, we updated the docs to mention that this method uses the version id from MiniAppInfo and also that it should only be used in preview mode. See here: https://github.com/rakutentech/android-miniapp/blob/master/miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniApp.kt#L54

Maybe we add something similar here?

Reply:
sure you're right